.travis.yml

```diff
@@ -67,6 +68,21 @@ matrix:
     python: 3.8
     dist: bionic
 
+  # Build and deploy the aarch64 wheels on tagged releases to master branch
+  - stage: deploy
+    arch: arm64
+    dist: bionic
+    python: 3.8
+    services:
+      - docker
+    if: (branch IS master) AND (tag IS present) AND (repo = python-greenlet/greenlet)
+    install:
+      - python3 -m pip install twine
+    script:
+      - ./make-manylinux manylinux2014_aarch64
+    after_succes:
```

Comment:
Is it maybe a typo? Shouldn't it be `after_success`?

Reply:
Yes, that's a typo.  I'll fix that.

Reply:
fixed